app/controllers/events_controller.rb

```diff
@@ -60,7 +60,7 @@ def featured
     @events = Event.includes(:neighborhood).near_user(current_location).featured.upcoming
     @current_sponsor = current_sponsor
 
-    if @events.featured.count == 0
+    if @events.featured.count(:all) == 0
```

Comment:
Checking .empty? is better than count() == 0.  count() may not fast, while checking .empty() is fast